src/Microsoft.PowerShell.Commands.Utility/commands/utility/Select-Object.cs

```diff
@@ -607,7 +607,7 @@ private void FilteredWriteObject(PSObject obj, List<PSNoteProperty> addedNotePro
             {
```

Comment:
@iSazonov, your last commit had 1 failures in `PowerShell-CI-windows`
Invoke-WebRequest tests.Denial of service.Charset Parsing
```powershell
Expected the actual value to be greater than 10, but got 7.14963570162819.
at <ScriptBlock>, D:\a\1\s\test\powershell\Modules\Microsoft.PowerShell.Utility\WebCmdlets.Tests.ps1: line 1959
1959:             $pathologicalRatio | Should -BeGreaterThan 10
```